core/src/main/kotlin/no/strooped/view/screen/components/PlacementLabel.kt

```diff
@@ -0,0 +1,54 @@
+package no.strooped.view.screen.components
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.g2d.BitmapFont
+import com.badlogic.gdx.math.Vector2
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+
+class PlacementLabel(
+    val placement: Int,
+    position: Vector2,
+    width: Float,
+    font: Float
+) : Label("", LabelStyle(BitmapFont(Gdx.files.internal("applegothic.fnt")), Color.WHITE)) {
+    init {
+        var textToDisplay = placement.toString()
+        @Suppress("MagicNumber") run {
+            color = when (placement) {
+                1 -> Color.GOLD
+                2 -> Color.LIGHT_GRAY
+                3 -> Color.BROWN
+                else -> {
+                    Color.RED
+                }
+            }
+            textToDisplay += if (placement > 10 && ((placement % 10) in (1..3)) && ((placement / 10) % 10 == 1)) {
```

Comment:
I suggest creating a `pluralize`-function, and use that here instead.

```kotlin
// no.tdt4240.strooped.util.pluralizeUtil.kt

fun String.pluralize(count): String {
  val suffix = when (count) {
    1 -> "st"
    2 -> "nd"
    3 -> "rd"
    else -> "th"
  }

  return "$this$suffix"
}

// ...
// PlacementlLabel.kt

// I think using placement directly is correct? Since position 12, shouldn't give '12nd' but instead '12th'
val placementFormatted = placement.toString().pluralize(placement)
val textToDisplay = "$placementFormatted place"
```

Reply:
yea but 21 should be 21st for example..that is why I added the logic in the first if

Reply:
Yeah, but i would then add it as an edge case inside the pluralize function. You could potentially hook the pluralize function onto the `Int` type instead of `String`

Reply:
done